challenges/binary-search/test_binary_search.py

```diff
@@ -3,36 +3,21 @@
 import binary_search
 
 
-def test_sorted_array_is_empty():
+def test_array_if_empty(sorted_array, test_value, low, high):
     """
-    test if sorted array is empty
+    Test check cases wheere array are empty
     """
     sorted_array = []
-    test_value = 2
-    assert sorted_array(sorted_array, test_value)
+    test_value = 25
+    assert binary_search.sub_binary_search(sorted_array, test_value, low,
```

Comment:
You generally do not need to test the recursing function, just the base function (`binary_search1`).